src/Symfony/Bundle/FrameworkBundle/DependencyInjection/FrameworkExtension.php

```diff
@@ -1859,22 +1865,20 @@ private function registerCacheConfiguration(array $config, ContainerBuilder $con
                     ->setPublic($pool['public'])
                 ;
 
-                $pool['name'] = $name;
+                $pool['name'] = $tagAwareId = $name;
                 $pool['public'] = false;
                 $name = '.'.$name.'.inner';
-
-                if (!\in_array($pool['name'], ['cache.app', 'cache.system'], true)) {
-                    $container->registerAliasForArgument($pool['name'], TagAwareCacheInterface::class);
-                    $container->registerAliasForArgument($name, CacheInterface::class, $pool['name']);
-                    $container->registerAliasForArgument($name, CacheItemPoolInterface::class, $pool['name']);
-                }
             } elseif (!\in_array($name, ['cache.app', 'cache.system'], true)) {
                 $container->register('.'.$name.'.taggable', TagAwareAdapter::class)
```

Comment:
You can use $tagAwareId here if you define it just before

Reply:
Good catch, thank you. Fixed.